src/frontend/screens/Login/index.tsx

```diff
@@ -47,33 +53,47 @@ export default function NewLogin() {
       <div className="loginBackground"></div>
 
       <div className="loginContentWrapper">
-        {!loading && (
-          <LanguageSelector
-            flagPossition={FlagPosition.PREPEND}
-            showWeblateLink={true}
-          />
-        )}
         <div className="runnerList">
-          <Runner
-            class="epic"
-            loginUrl="/loginweb/legendary"
-            icon={() => <EpicLogo />}
-            isLoggedIn={isEpicLoggedIn}
-            user={epic.username}
-            logoutAction={epic.logout}
-            alternativeLoginAction={() => {
-              setShowSidLogin(true)
-            }}
-          />
-          <Runner
-            class="gog"
-            icon={() => <GOGLogo />}
-            loginUrl="/loginweb/gog"
-            isLoggedIn={isGogLoggedIn}
-            user={gog.username}
-            logoutAction={gog.logout}
-          />
+          <div className="runnerHeader">
+            <HeroicLogo className="runnerHeaderIcon" />
+            <div className="runnerHeaderText">
+              <h1 className="title">Heroic</h1>
+              <h2 className="subtitle">Games Launcher</h2>
+            </div>
+
+            {!loading && (
+              <LanguageSelector
+                flagPossition={FlagPosition.PREPEND}
+                showWeblateLink={true}
+              />
+            )}
+          </div>
+
+          <p className="runnerMessage">{loginMessage}</p>
+
+          <div className="runnerGroup">
```

Comment:
We don't need  this `runnerGroup` wrapper div, having the runners directly after the `p` works fine (they just need `margin-bottom: var(--login-padding)`).

Reply:
hmmm checking the CSS I think this is added here for the future if we add more stores?

Reply:
> hmmm checking the CSS I think this is added here for the future if we add more stores?

Yeah that's the goal, it should look something like pic attached (just duplicated gog login for demo). Although it can be removed for another PR down the line it's kind of not useful right now. 
<img width="886" alt="Screenshot 2022-10-22 at 4 50 09 PM" src="https://user-images.githubusercontent.com/99497246/197361917-5b5285d9-1c9c-40e4-8465-d43167eec9d3.png">